### PR TITLE
Number of employees as a setting

### PR DESCRIPTION
**Pull request by gcichosz**
# Description

Change hard-coded number of employees value, to a setting.

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Manually ;) Debug Http_TotalNumbers function and check if it initializes EmployeesEngagementCalculator with correct number of employees value.

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/BurnForMoney.Functions.PublicApi/Configuration/ApplicationConfiguration.cs [hunk 1]
```diff
@@ -18,11 +18,16 @@ public static ConfigurationRoot GetSettings(string functionAppDirectory)
...
                 _settings = new ConfigurationRoot
                 {
                     ConnectionStrings = new ConnectionStringsSection
                     {
                         SqlDbConnectionString = config["ConnectionStrings:Sql"]
+                    },
+                    CompanyInformation = new CompanyInformationSection
+                    {
+                        NumberOfEmployees = numberOfEmployees
                     }
```
Comment (pmaga) [inline src/BurnForMoney.Functions.PublicApi/Configuration/ApplicationConfiguration.cs]: you don't need to parse it manually. As long as class model reflects app settings structure you can use write it in single line: CompanyInformation = config.Get< CompanyInformationSection>("CompanyInformation")

File: src/ResourceGroup/Template.json [hunk 2]
```diff
@@ -82,7 +82,8 @@
     "keyVaultName": "[concat(variables('applicationName'), 'kv', parameters('environmentCode'))]",
     "stravaAccessTokensKeyVaultName": "[concat(variables('applicationShortName'), 'kvstravatokens', parameters('environmentCode'))]",
     "appInsightsName": "[concat(variables('applicationName'), 'ai', parameters('environmentCode'))]",
-    "eventGridTopicName": "[concat(variables('applicationShortName'), 'domainEvents', parameters('environmentCode'))]"
+    "eventGridTopicName": "[concat(variables('applicationShortName'), 'domainEvents', parameters('environmentCode'))]",
+    "numberOfEmployees": "100"
```
Comment (pmaga) [inline src/ResourceGroup/Template.json]: you can use integers here

File: src/BurnForMoney.Functions.PublicApi/Configuration/ApplicationConfiguration.cs (hunk 1 above)
Comment (gcichosz) [reply src/BurnForMoney.Functions.PublicApi/Configuration/ApplicationConfiguration.cs]: Good to know, although you probably meant `config.GetSection("CompanyInformation").Get<CompanyInformationSection>()`. Fixed.

File: src/ResourceGroup/Template.json (hunk 2 above)
Comment (gcichosz) [reply src/ResourceGroup/Template.json]: True, is'a a JSON file... Fixed.

**Pull request merged**